src/ui/cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
Do we need this file for now? Not a fan of adding in a bunch of commented-out code/examples.